### PR TITLE
fix: trigger action strings for trigger_type and flow_type

### PR DESCRIPTION
**Pull request by stebenz**

### CODE REVIEW

File: zitadel/v2/trigger_actions/resource.go [hunk 1]
```diff
@@ -42,3 +52,22 @@ func GetResource() *schema.Resource {
...
+	return map[int32]string{
+		0: "FLOW_TYPE_UNSPECIFIED",
+		1: "FLOW_TYPE_EXTERNAL_AUTHENTICATION",
+		2: "FLOW_TYPE_CUSTOMISE_TOKEN",
+		3: "FLOW_TYPE_INTERNAL_AUTHENTICATION",
+	}
+}
+func triggerTypes() map[int32]string {
+	return map[int32]string{
+		0: "TRIGGER_TYPE_UNSPECIFIED",
```
Comment (eliobischof) [inline zitadel/v2/trigger_actions/resource.go:66]: Let's remove TRIGGER_TYPE_UNSPECIFIED

File: zitadel/v2/trigger_actions/resource.go [hunk 2]
```diff
@@ -42,3 +52,22 @@ func GetResource() *schema.Resource {
 		Importer:      &schema.ResourceImporter{StateContext: schema.ImportStatePassthroughContext},
 	}
 }
+
+func flowTypes() map[int32]string {
+	return map[int32]string{
+		0: "FLOW_TYPE_UNSPECIFIED",
```
Comment (eliobischof) [inline zitadel/v2/trigger_actions/resource.go:58]: Let's remove FLOW_TYPE_UNSPECIFIED

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: zitadel/v2/trigger_actions/resource.go (hunk 1 above)
Comment (stebenz) [reply zitadel/v2/trigger_actions/resource.go:66]: Agreed.

File: zitadel/v2/trigger_actions/resource.go (hunk 2 above)
Comment (stebenz) [reply zitadel/v2/trigger_actions/resource.go:58]: Agreed.

**Review by eliobischof — APPROVED** [review]

**Pull request merged**